libkbfs/md_ops_test.go

```diff
@@ -195,12 +206,16 @@ func expectGetKeyBundles(ctx context.Context, config *ConfigMock, extra ExtraMet
 	if extraV3, ok := extra.(*ExtraMetadataV3); ok {
 		config.mockMdserv.EXPECT().GetKeyBundles(
 			ctx, gomock.Any(), gomock.Any(), gomock.Any()).
-			Return(extraV3.wkb, extraV3.rkb, nil)
+			Return(&extraV3.wkb, &extraV3.rkb, nil)
 	}
 }
 
 func TestMDOpsGetForHandlePrivateSuccess(t *testing.T) {
-	mockCtrl, config, ctx := mdOpsInit(t)
+	runTestOverMetadataVers(t, testMDOpsGetForHandlePrivateSuccess)
```

Comment:
For all these tests, I think it might be cleaner if you do something like:
```go
func TestMDOps(t *testing.T) {
    tests := []func(*testing.T, MetadataVer) {
        testMDOpsGetForHandlePrivateSuccess,
        testMDOpsGetForUnresolvedHandlePublicSuccess,
        ...
    }
    for _, f := range tests {
        name := runtime.FuncForPC(reflect.ValueOf(f).Pointer()).Name()
        test := func(t2 *testing.T) {
            runTestOverMetadataVers(t2, f)
        }
        t.Run(name, test)
    }
}
```
The `-run`/`-test.run` flag filtering now gets changed to `-run "/testMDOpsGetForHandlePrivateSuccess"`

You can even parallelize them if you like with a `t.Parallel()` in the test func.

https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks